iphone/Classes/TiContactsPerson.m

```diff
@@ -84,33 +84,26 @@ -(NSDictionary*)getiOS9ContactProperties: (CNMutableContact*) contact
 	if (contact == nil) {
 		return nil;
 	}
-	return [[NSDictionary alloc] initWithObjectsAndKeys:contact.givenName, @"firstName",
-		contact.familyName, @"lastName",
-		contact.middleName, @"middleName",
-		contact.namePrefix, @"prefix",
-		contact.nameSuffix, @"suffix",
-		contact.nickname, @"nickname",
-		contact.phoneticGivenName, @"firstPhonetic",
-		contact.phoneticFamilyName, @"lastPhonetic",
-		contact.phoneticMiddleName, @"middlePhonetic",
-		contact.organizationName, @"organization",
-		contact.jobTitle, @"jobTitle",
-		contact.departmentName, @"department",
-		contact.emailAddresses, @"email",
-		contact.note, @"note",
-//		contact.birthday, @"birthday", birthday property has some problems in iOS9 Beta, have to be exclusively handled
```

Comment:
Birthdays seems to have a strange issue. We are getting the birthday from selected CNContact. But CNContactPropery.key is returning "birthdays" with suffix s. This is tossing the lookup and returning the birthday as empty. Not sure if this is an issue with iOS 9. 

iOS 8 works as expected.